webrtc.html

```diff
@@ -2840,6 +2898,137 @@
         </dl>
       </section>
 
+      <section>
+        <h3>RTCDtlsTransport Interface</h3>
+
+        <p>The <code>RTCDtlsTransport</code> interface allows an
+        application access to information about the Datagram Transport
+        Layer Security (DTLS) transport over which RTP and RTCP
+        packets are sent and received by <code>RTCRtpSender</code>
+        and <code>RTCRtpReceiver</code> objects, as well other data
+        such as SCTP packets sent and received by data channels.  In
+        particular, DTLS adds security to an underlying transport, and
+        the <code>RTCDtlsTransport</code> interface allows access to
+        information about the underlying transport and the security
+        added.
+
+        <dl class="idl" title=
+          "interface RTCDtlsTransport">
+          <dt>readonly attribute RTCIceTransport transport</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-dtlstransport-transport"><code>RTCDtlsTransport.transport</code></dfn>
+            attribute is the underlying transport that is used to send
+            and receive packets.  The underlying transport may not be
+            shared between multiple <code>RTCDtlsTransport</code>
+            objects.
+          </dd>
+
+          <dt>readonly attribute RTCDtlsTransportState state</dt>
+
+          <dd>
+            <p>The <code id="dom-dtlstransport-state">state</code>
+            attribute MUST return the state of the transport.
+          </dd>
+
+          <dt>sequence&lt;ArrayBuffer&gt; getRemoteCertificates()</dt>
+
+          <dd>
+            <p>An array containing the remote certificates in use by
+            the remote side.</p>
+          </dd>
+
+          <dt>attribute EventHandler onstatechange</dt>
+
+          <dd>
+            This event handler, of event handler event type <code><a href=
+            "#event-dtlstransportstatechange">dtlstransportstatechange</a></code>,
```

Comment:
I sent an email to the list asking what name the WG would prefer (state, readystate, orconnectionstate).  I don't feel too strongly about it, but I thought it would be better for it to be consensus before changing it in the PR.